spooky-doodle/Bangazon-Platform-API
# Departments endpoint

# Description

Implemented Departments endpoint with all required functionality.

Fixes #6 

## Type of change

Please delete options that are not relevant.


# Testing Instructions

1.  Run testing_bangazon.sql to refresh the test database.
2.  Run all tests (all should pass).




# Checklist:
